types/public/README.md

```diff
@@ -0,0 +1,41 @@
+# Type definitions for Azure Functions
+These are type definitions for using TypeScript with Azure Functions.
+
+Although Azure Functions does not directly support functions written in TypeScript, running transpiled functions is supported. Read more [here](https://docs.microsoft.com/azure/azure-functions/functions-reference-node#configure-function-entry-point) on configuring the location and name of your Azure Function. 
```

Comment:
This is probably fine for now, but you're framing this in a fairly negative light. Once we have better tooling support I'd change to something like "We support typescript development, but we don't automatically transpile and run TS files"